crates/task-impls/src/transactions.rs

```diff
@@ -456,10 +443,14 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, Ver: StaticVersionType>
                 }
             };
 
-            let (block, header_input) = futures::join! {
-                client.claim_block(block_info.block_hash.clone(), view_number.u64(), self.public_key.clone(), &request_signature),
-                client.claim_block_header_input(block_info.block_hash.clone(), view_number.u64(), self.public_key.clone(), &request_signature)
-            };
+            let block = client
+                .claim_block(
```

Comment:
Something seems up with the spacing here.

Reply:
Yeah, it looks weird, but that's what rustfmt says it should look like for some reason 🤷